products/compute/api.yaml

```diff
@@ -11316,6 +11316,61 @@ objects:
           PARTNER InterconnectAttachment is created, updated,
           or deleted.
         output: true
+      - !ruby/object:Api::Type::Array
+        name: bfd
+        description: |
+          BFD configuration for the BGP peering.
+        min_version: beta
+        item_type: !ruby/object:Api::Type::NestedObject
+            properties:
```

Comment:
We don't _love_ objects with all optional fields, because it lets people have empty blocks that are not super intuitive to use (this was a big push of ours for 3.0.0). Can one of these fields be made Required? (you could also use at_least_one_of here, but since they all have defaults it makes more sense to me to choose one to be required)

Reply:
sessionInitializationMode is the one that makes more sense to be explicitly provided, so now marked as required (as long as bfd itself is passed).